src/Motions/Methods.ts

```diff
@@ -0,0 +1,129 @@
+import { window, Position, commands, SymbolInformation, SymbolKind, Location, TextDocument } from 'vscode';
+import { Motion } from './Motion';
+
+enum Direction { Prev, Next }
+
+export class MotionMethod extends Motion {
+
+    private direction: Direction;
+    private n: number;
+
+    constructor(args: { direction: Direction, n?: number }) {
+        args.n = args.n === undefined ? 1 : args.n;
```

Comment:
Could be solved a bit cleaner with [nullish coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing), assuming you're using TS 3.7+.

```typescript
args.n = args.n ?? 1;
```